### ISSUE TITLE
lxml pinning necessary?

### ISSUE THREAD

**Issue by hoechenberger** [BEL-Public/mffpy#105]
Hello, I was a bit surprised you're pinning lxml to a very specific version, not even allowing for newer patch releases:

https://github.com/BEL-Public/mffpy/blob/3674d76008e1a4551ee0254f654972c4a9df4449/requirements.txt#L1

Is this intentional or maybe just a typo and you actually wanted to specify `>= 4.8.0` or `~= 4.8.0`?

Thanks,
Richard

**Comment by mscheltienne** [comment]
I have a package recipe that depends on MNE that is failing because conda is installing the wrong version of `lxml`:

```
mffpy 0.7.3 has requirement lxml==4.8.0, but you have lxml 4.9.0.
```

c.f. https://github.com/conda-forge/staged-recipes/pull/19298 and [CI run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=522876&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823).

For now, I will drop the req. from `mne` to `mne-base`, but it kind of looks related to the change here. I still have no idea why conda is retrieving the wrong version of `lxml`, but maybe this change broke a couple of packages depending on `mne` or on `mffpy` feedstocks?

**Comment by hoechenberger** [comment]
@mscheltienne this really should happen as I've pinned lxml to 4.8 for mffpy and MNE depends on mffpy and hence should fetch the correct version of lxml 🤔 or does your package explicitly depend on lxml 4.9?

**Comment by hoechenberger** [comment]
@mscheltienne also feel free ti make a PR to mffpy-feedstock, remove the pinning and the pip check test. Then it should all work.

**Comment by mscheltienne** [comment]
No, this package does not depend on `lxml`. The dependencies are fairly simple, and can be summarized as `mne-base` + `joblib` + `scikit-learn`. I agree that this should not happen and `conda` should correctly fetch the 4.8 version.. but that's not the case. I wonder if this pin silently broke other recipes and feedstocks..

Let's remove the pin from both the feedstock and from the requirements? Or was there a specific reason that @damian5710 you included this version pinning in #101?

**Comment by ephathaway** [comment]
@hoechenberger, @mscheltienne, this is fixed in `v0.7.4`. Here the `lxml` dependency is `>=4.8.0`.
